### ISSUE TITLE
Integrate selection of approved bidders using their deposits to Frontend CLI

### ISSUE THREAD

**Issue by uhbif19** [mlabs-haskell/hydra-auction#258]
Todo:

- [x] Function to find current deposits on L1 (just query by Address)
- [x] Change Frontend CLI to support selection by deposits (UX TBD)

Blocked by #257 
Blocked by #222 
Blocked by #221 
Blocked by #295

**Comment by uhbif19** [comment]
@nazrhom Moved to M6, cuz all it depends on `AuctionBidders` logic, disabled up to M6.

**Comment by nazrhom** [comment]
We discussed this a few meetings ago: @GeorgeFlerovsky mentioned that having a query to filter all users that have deposited an amount that is >= n for some n.

**Comment by uhbif19** [comment]
This is following point in todo list:

>  Change Frontend CLI to support selection by deposits (UX TBD)

Left is "Publish bidder approval on Platform server", I will separate it to another task.
